homeassistant/components/androidtv/strings.json

```diff
@@ -32,12 +32,12 @@
         "title": "Android TV Options",
         "data": {
           "apps": "Configure applications list",
-          "get_sources": "Whether or not to retrieve the running apps as the list of sources",
-          "exclude_unnamed_apps": "Exclude app with unknown name",
-          "screencap": "Determines if album art should be pulled from what is shown on screen",
+          "get_sources": "Use the running apps as the list of sources",
+          "exclude_unnamed_apps": "Exclude apps with unknown name",
```

Comment:
If `exclude_unnamed_apps` is true, then only apps configured in the `apps` dictionary will be listed among the sources. I'm not sure whether the current app is included even if it isn't configured in the `apps` dictionary. 

In general, the handling of apps as sources has gotten a bit messy and could use some cleaning up.

Reply:
Here I think that I can leave as it is and again use your explanation for documentation

Reply:
Yeah, it's probably OK as it is.